msteams-platform/index-test.yml

```
@@ -0,0 +1,171 @@
+          text: Build a Microsoft Teams app
+        - url: /microsoftteams/platform/concepts/extensibility-points
+          itemType: concept
+          text: Capability points in Microsoft Teams
+        - url: /microsoftteams/platform/concepts/design/understand-use-cases
+          itemType: concept
+          text: Understand Microsoft Teams use cases
+        - url:  /microsoftteams/platform/concepts/design/map-use-cases
+          itemType: concept
+          text: Map use cases to Microsoft Teams capabilities
+        - url:  /microsoftteams/platform/whats-new 
+          itemType: whats-new
+          text: What's new
+      
+    # Card
+    - title: Learn
+      links:
+        - url: /learn/modules/intro-microsoft-teams-apps
+          itemType: learn
+          text: Build apps for Microsoft Teams — introduction
+        - url: /learn/modules/embedded-web-experiences
+          itemType: learn
+          text: Tabs — create embedded web experiences 
+        - url:  /learn/modules/msteams-conversation-bots/
+          itemType: learn
+          text: Bots — create conversational interactions
+        - url: /learn/modules/msteams-messaging-extensions/
+          itemType: learn
+          text: Messaging extensions — create task-oriented interactions 
+        - url: /learn/modules/msteams-task-modules/
+          itemType: learn
+          text: Task modules — collect input and gather information
+        - url: /learn/modules/msteams-webhooks-connectors/
+          itemType: learn
+          text: Webhooks and Office 365 connectors — connect web services to channels and teams
+    # Card
+    - title: Develop
+      links:
+        - url: /microsoftteams/platform/tutorials/get-started-nodejs-app-studio
+          itemType: get-started
+          text: Build a Teams app using JavaScript/Node.js and App Studio
+        - url: /microsoftteams/platform/tutorials/get-started-dotnet-app-studio
+          itemType: get-started
+          text: Build a Teams app using C#/.NET and App Studio
+     # Card
+    - title: Deploy
+      links:
+        - url: /microsoftteams/platform/samples/app-templates
+          itemType: deploy
+          text: Teams app templates — production-ready applications
+        - url:  /microsoftteams/platform/samples/shifts-wfm-connectors
+          itemType: deploy
+          text: Teams shifts connectors - production-ready integrations
+        - url: /microsoftteams/platform/bots/how-to/add-power-virtual-agents-bot-to-teams
+          itemType: deploy
+          text: Integrate your Power Virtual Agents bot with Teams     
+    # Card
+    - title: Connect
+      links:
+        - url: https://developer.microsoft.com/microsoft-teams/blogs/
+          itemType: overview
+          text: Microsoft Teams developer blog
+        - url: https://aka.ms/microsoftteamscommunitycall
+          itemType: overview
+          text: Microsoft Teams Community Calls
+    # Card
+    - title: Community Help
```

```suggestion
    - title: Community help
```
Changed to sentence case per our style.